service/service_test.go

```diff
@@ -40,8 +40,10 @@ func Test_Service_New(t *testing.T) {
 				config.Source = "test"
 
 				config.Viper.Set(config.Flag.Guest.Cluster.Kubernetes.API.ClusterIPRange, "172.31.0.0/16")
+				config.Viper.Set(config.Flag.Guest.Cluster.Network.CIDR, "172.23.0.0/16")
```

Comment:
What is the purpose of the default value here? What happens if its used will the cluster work but peering fail?

Reply:
This is a unit test to verify that service instantiation succeeds. This test has often caught mistakes in missing dependencies.